jck/build.xml

```diff
@@ -68,10 +66,18 @@
 			</then>
 			<!-- jck materials exist, update jck materials if needed-->
 			<else>
+				<echo message="Running git clean at ${JCK_ROOT_USED}..." />
+				<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="true">
+					<arg value="clean" />
+					<arg value="--force" />
+					<arg value="--quiet" />
+				</exec>
+				
 				<echo message="${JCK_ROOT_USED} exists, deleting previously built natives..." />
 				<delete includeemptydirs="true" quiet="true" failonerror="false">
 				    <fileset dir="${JCK_ROOT_USED}/natives" includes="**/*"/>
 				</delete>
+								
 				<echo message="Updating ${JCK_ROOT_USED} with latest..." />
 				<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="true">
 					<arg value="pull" />
```

Comment:
This `pull` has `origin` and `master` hardcoded. This may someday cause an issue. You may be able to do a `git pull` by itself which will assume your remote and branch. You could alternatively replace `master` with `${jck_branch}`.

Reply:
Removed origin and master. Seems to work fine. Tested here: Test_grinder/job/Grinder/3099/console